### ISSUE TITLE
Losing Items Bought With Credits.

### ISSUE THREAD

**Issue by Upsoar** [Avicus/Issues#603]
Since the removal of the 100000 credit reward, I've been losing things that I have bought/buy with my credits. I've LOST about 10k credits because of this glitch and I'd like to see it fixed. In addition, I'd like to know if there is in fact a credit reward for the tournament, because I have not received and credit reward.

Me attempting to apply colors I had bought to my badge, replaces each color with red:
http://gyazo.com/0026b5f2077c416316524b34bdd2f7d7

**Comment by KyloCode** [comment]
Anything past the time of the credit issue was removed for those specific players, I've got logs of it to check, but there shouldn't be any issue with loss of credits, that was made a priority. There is a credit reward and we'll probably get that out in the next 24 hours. Sorry for the issues, and apologies for the actions of certain staff not acting on behalf of the staff as a whole.

I don't understand what's being shown in the image. Looks like a backpack of a few items.

**Comment by Upsoar** [comment]
Everything in that backpack is supposed to be a different color. I have the red currently equipped on my badge, and every time I tried to select a new color (yellow for example) it would be replaced with the red wool without changed my badge to yellow. So instead of having 4 different colors, this picture shows me having 2. And I know this isn't just a visual glitch, because it's still like this. I can currently only switch between Red and Purple. My other colors I owned no longer exist.
